src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -477,7 +499,7 @@ uint256 AsyncRPCOperation_sendmany::main_impl() {
             },
             [&](const libzcash::SaplingPaymentAddress& addr) {
                 auto value = r.amount;
-                auto memo = get_memo_from_hex_string(r.memo.has_value() ? r.memo.value() : "");
+                auto memo = get_memo_from_hex_string(r.memo.value_or(""));
```

Comment:
Aren't we supposed to use 0xF6 || 511 zero bytes for an empty memo?

Reply:
This "works" because of the way `AsyncRPCOperation_sendmany::get_memo_from_hex_string` is defined. I'm looking at this code now and I do not like it, but AFAICT it correctly manages to convert the empty string into the no-memo value.